docusaurus.config.js

```diff
@@ -187,7 +187,10 @@ module.exports = {
       indexName: 'temporal',
       // contextualSearch: true, // Optional, If you different version of docs etc (v1 and v2) doesn't display dup results
       // appId: 'app-id', // Optional, if you run the DocSearch crawler on your own
-      // algoliaOptions: {}, // Optional, if provided by Algolia
+      searchParameters: { // used to be called algoliaOptions https://docusaurus.io/docs/search#using-algolia-docsearch
+        facetFilters: [],
+        facets: []
```

Comment:
this does not work to clear the facetFilters field. i also tried `facetFilters: null` and that did not work.

Reply:
somehow the legacy search page does not respect these settings, it's like contextual search is always enabled by default in this page so you can't remove the hardcoded facet filters.

But this highlighted a more important problem: your docsearch config is wrong. Fix it, and contextual search + facet filters will both work.